src/WebJobs.Extensions.DurableTask/Listener/DurableTaskTriggerMetrics.cs

```diff
@@ -16,7 +17,7 @@ internal class DurableTaskTriggerMetrics : ScaleMetrics
         /// The number of messages across control queues. This will
         /// be in the form of a serialized array of ints, e.g. "[1,2,3,4]".
         /// </summary>
-        public string ControlQueueLengths { get; set; }
+        public virtual string ControlQueueLengths { get; set; }

```

Comment:
made this virtual to enable mocking. Same with `WorkItemQueueLength=

Reply:
I suggest we just make all of them virtual for consistency, unless you think there's a reason not to.

Reply:
Fair enough, they're all virtual now :)